packages/tools/src/helpers.ts

```diff
@@ -196,6 +196,7 @@ export const getSupportedBundlers = (getPlugins: GetPlugins) => {
         },
         context: getContext({
             // We don't care, this is a hack.
```

Comment:
x)

Reply:
😄 brutally honest
(also, this is internal only)